run_scripts/1_dadaab_coda_get.sh

```diff
@@ -55,6 +61,8 @@ DATASETS=(
     "s02_dadaab_impact_made"
 )
 
+./log_pipeline_event.sh "$USER" "$GOOGLE_CLOUD_CREDENTIALS_FILE_PATH" "$PIPELINE_CONFIGURATION" "$TIMESTAMP" "$RUN_ID" "1"
```

Comment:
I think this should be done in run_pipeline.sh rather than here because:
- This means we can run coda get by itself in development without needing to log an event anywhere.
- Logging a pipeline run only makes sense when you're running the whole pipeline - which (usually) happens only when calling run_pipeline.sh